test/statsmodel.jl

```diff
@@ -233,6 +235,13 @@ Base.show(io::IO, m::DummyModTwo) = println(io, m.msg)
     m2 = fit(DummyModTwo, f, d)
     # make sure show() still works when there is no coeftable method
     show(io, m2)
+
+    # one final termnames check
+    # note that `1` is still a ConstantTerm and not yet InterceptTerm
+    # because apply_schema hasn't been called
```

Comment:
Maybe this is worth mentioning in the docstring?

Reply:
done